docs/current-1.1/contributing.md

```diff
@@ -0,0 +1,60 @@
+# Using the current-1.1 folder and sub-folders
+
+This folder is the top-level node that matches the [docs](../docs)
```

Comment:
This link leads to `docs/docs`. Is the link wrong, or the location of the file (`docs/current-1.1`)?